EditorExtensions/Shared/Compilers/ServerBase.cs

```diff
@@ -92,8 +91,11 @@ private void Dispose(bool disposing)
             {
                 using (await mutex.LockAsync())
                 {
-                    if (server == null || server._process == null || server._process.HasExited)
+                    if (server == null || server._address == null || server._process == null || server._process.HasExited)
+                    {
                         server = new T();
+                        server._address = string.Format(CultureInfo.InvariantCulture, "http://localhost.:{0}/", server.BasePort);
```

Comment:
It has been a while but if I remember it correctly, previously it was blocking the folks behind corporate proxy and using Windows 7 or Server 2012, where user has IE8 installed. So adding the dot helps.

Note that this is the negotiation time, when we turn up the node.js server for WebEssentials2013. We try to reach the server with `localhost.`, attempt five times, then fallback to `localhost`, then reattempt five more times with that and finally fallback to `127.0.0.1`. If none of the above work, we abort the node.js process and log the message in Output Window logger. Otherwise, we use the successful address for the lifetime of the given VS instance.